index.js

```diff
@@ -1,17 +1,43 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
 
-  add(item) {}
+  }
 
-  get(pos) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  max() {}
+  get(pos) {
+    if (pos < 0 || pos >= this.length) {
+      throw new Error('OutOfBounds');
+    }
+    return this.items[pos];
+  }
 
-  min() {}
 
-  sum() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error('EmptySortedList');
+    }
+    return this.items[this.length - 1];
```

Comment:
Tambien aqui lo que podemos hacer es utilizar la función max() retorna el valor máximo presente en la lista. Para hacer esto 🔥 , utiliza el método Math.max(...this.items), que toma una cantidad variable de argumentos y devuelve el valor más grande entre ellos. En este caso, el operador de propagación ... se utiliza para pasar los elementos individuales de la lista como argumentos a la función Math.max().
```
 return Math.max(...this.items)
```